packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -19,9 +19,35 @@ const meta: ComponentMeta<typeof IconButton> = {
       page: mdx,
     },
   },
+  args: {
+    name: 'apps',
+    size: 'xxxl',
+    iconType: 'default',
+    tooltip: 'sample tooltip message',
+    tooltipPlacement: 'top',
+    variant: 'primary',
```

Comment:
Same here about the default being `'secondary'`

Reply:
done!